### PR TITLE
CRUD for whitelisted urls

### PR DESCRIPTION
**Pull request by emilsoman**

### CODE REVIEW

File: apps/dockup_ui/config/prod.exs [hunk 1]
```diff
@@ -14,7 +14,7 @@ use Mix.Config
 config :dockup_ui, DockupUi.Endpoint,
   http: [port: {:system, "PORT"}],
   url: [host: (System.get_env("DOCKUP_DOMAIN") || "example.com"), port: (System.get_env("DOCKUP_PORT") || 80)],
-  cache_static_manifest: "priv/static/manifest.json"
+  cache_static_manifest: "priv/static/cache_manifest.json"
```
Comment (emilsoman) [inline apps/dockup_ui/config/prod.exs]: This is for phoenix 1.3 . Unrelated to whitelist url related changes

File: apps/dockup_ui/test/controllers/whitelisted_url_controller_test.exs [hunk 2]
```diff
@@ -0,0 +1,66 @@
+defmodule DockupUi.WhitelistedUrlControllerTest do
```
Comment (emilsoman) [inline apps/dockup_ui/test/controllers/whitelisted_url_controller_test.exs]: This file is autogenerated content

File: apps/dockup_ui/test/models/whitelisted_url_test.exs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+defmodule DockupUi.WhitelistedUrlTest do
```
Comment (emilsoman) [inline apps/dockup_ui/test/models/whitelisted_url_test.exs]: This file is autogenerated content

File: apps/dockup_ui/web/controllers/whitelisted_url_controller.ex [hunk 4]
```diff
@@ -0,0 +1,65 @@
+defmodule DockupUi.WhitelistedUrlController do
```
Comment (emilsoman) [inline apps/dockup_ui/web/controllers/whitelisted_url_controller.ex]: This file is autogenerated content

File: apps/dockup_ui/web/models/whitelisted_url.ex [hunk 5]
```diff
@@ -0,0 +1,18 @@
+defmodule DockupUi.WhitelistedUrl do
```
Comment (emilsoman) [inline apps/dockup_ui/web/models/whitelisted_url.ex]: This file is autogenerated content

File: apps/dockup_ui/web/templates/whitelisted_url/edit.html.eex [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<h2>Edit whitelisted url</h2>
```
Comment (emilsoman) [inline apps/dockup_ui/web/templates/whitelisted_url/edit.html.eex]: This file is autogenerated content

File: apps/dockup_ui/web/templates/whitelisted_url/form.html.eex [hunk 7]
```diff
@@ -0,0 +1,17 @@
+<%= form_for @changeset, @action, fn f -> %>
```
Comment (emilsoman) [inline apps/dockup_ui/web/templates/whitelisted_url/form.html.eex]: This file is autogenerated content

File: apps/dockup_ui/web/templates/whitelisted_url/index.html.eex [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<h2>Listing whitelisted urls</h2>
```
Comment (emilsoman) [inline apps/dockup_ui/web/templates/whitelisted_url/index.html.eex]: This file is autogenerated content

File: apps/dockup_ui/web/templates/whitelisted_url/new.html.eex [hunk 9]
```diff
@@ -0,0 +1,6 @@
+<h2>New whitelisted url</h2>
```
Comment (emilsoman) [inline apps/dockup_ui/web/templates/whitelisted_url/new.html.eex]: This file is autogenerated content

File: apps/dockup_ui/web/templates/whitelisted_url/show.html.eex [hunk 10]
```diff
@@ -0,0 +1,13 @@
+<h2>Show whitelisted url</h2>
```
Comment (emilsoman) [inline apps/dockup_ui/web/templates/whitelisted_url/show.html.eex]: This file is autogenerated content

File: apps/dockup_ui/web/views/whitelisted_url_view.ex [hunk 11]
```diff
@@ -0,0 +1,3 @@
+defmodule DockupUi.WhitelistedUrlView do
```
Comment (emilsoman) [inline apps/dockup_ui/web/views/whitelisted_url_view.ex]: This file is autogenerated content

**Comment by ganeshmannamal** [comment]
Looks good. Tested on local

**Pull request merged**